src/lib.rs

```diff
@@ -108,9 +137,13 @@ pub trait PolynomialCommitment<F: Field>: Sized {
     /// open the commitment to produce an evaluation proof.
     type CommitterKey: PCCommitterKey;
     /// The verifier key for the scheme; used to check an evaluation proof.
-    type VerifierKey: PCVerifierKey;
+    type VerifierKey: PCVerifierKey + Default;
```

Comment:
Hmm the default bound on the `VerifierKey` might be incorrect for `ipa_pc::VerifierKey`, because that will just make an empty vector.

Reply:
I removed the `Default` and likely I will make sure Marlin/PCD did not use it.